.cirrus.yml

```diff
@@ -196,41 +198,74 @@ conformance_task:
 
 
 integration_task:
-    name: "Integration $DISTRO_NV w/ $STORAGE_DRIVER"
+    name: "Integration $DISTRO_NV$RUNTIME_N w/ $STORAGE_DRIVER"
     alias: integration
-    only_if: *not_build_docs
-    depends_on: *smoke_vendor_cross
+    skip: *not_build_docs
+    depends_on: *smoke_vendor
 
     matrix:
         # VFS
+        - env:
+            DISTRO_NV: "${PRIOR_FEDORA_NAME}"
+            IMAGE_NAME: "${PRIOR_FEDORA_CACHE_IMAGE_NAME}"
```

Comment:
These should use the current Fedora images, otherwise it'll be a duplicate of the section at line 220.

Reply:
I think if you check HEAD you'll see that I've fixed this.  There's definitely no need to test with "PRIOR_FEDORA..." AFAIK.